config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :messages
```

Comment:
since we only have a create action. We should add only: [:create] so we don't crowd the routes